pkg/envinfo/mocks/LocalConfigProvider.go

```diff
@@ -0,0 +1,99 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+
+package mocks
```

Comment:
Mocks generated for 
https://github.com/openshift/odo/blob/a28e37838c5cbaa396f6675fe90a9986168165f5/pkg/envinfo/envinfo.go#L90-L97 
 using testify `mockery`